api.raml

```diff
@@ -0,0 +1,418 @@
+          "tokenGive": "0x62a56a4a2ef4d355d34d10fbf837e747504d38d4",
+          "amountGive": "30000",
+          "nonce": "1982976399",
+          "expires": 5629999,
+          "signature": {
+            "r": "0xfd7aa97d7bdf41ee188ab6db5ce6fcbd312e9f8d1932df9b446820a5a7f6ff4a",
+            "s": "0x37eab8f9e95629f4ede94ed7e38d14f8fbd6bde6fcc19d24dc2aaea21bd5eaa1",
+            "v": 28
+          }
+        }
+    responses:
+      200:
+        body:
+          example: |
+            {
+              "id": "3135123",
+              "account": "0x1307b8d863e0cfc147ad0953613f98bbdf95be41",
+              "contract": "0x1cE7AE555139c5EF5A57CC8d814a867ee6Ee33D8",
+              "tokenGet": "0x0000000000000000000000000000000000000000",
+              "tokenGetSymbol": "ETH",
+              "tokenGetDecimals": 18,
+              "amountGet": "747000000000000000",
+              "tokenGive": "0x62a56a4a2ef4d355d34d10fbf837e747504d38d4",
+              "tokenGiveSymbol": "PAYX",
+              "tokenGiveDecimals": 2,
+              "amountGive": "30000",
+              "nonce": "1982976399",
+              "expires": 5629999,
+              "signature": {
+                "r": "0xfd7aa97d7bdf41ee188ab6db5ce6fcbd312e9f8d1932df9b446820a5a7f6ff4a",
+                "s": "0x37eab8f9e95629f4ede94ed7e38d14f8fbd6bde6fcc19d24dc2aaea21bd5eaa1",
+                "v": 28
+              },
+              "availableVolume": "747000000000000000",
+              "created_at": "2018-05-08T21:30:17.399Z",
+              "updated_at": "2018-05-08T21:30:17.399Z"
+            }
+  /{id}:
+    get:
+      description: Provide infomation for single order including not active order.
+      responses:
+        200:
+          body:
+            example: |
+                {
+                  "id": "3135123",
+                  "account": "0x1307b8d863e0cfc147ad0953613f98bbdf95be41",
+                  "contract": "0x1cE7AE555139c5EF5A57CC8d814a867ee6Ee33D8",
+                  "tokenGet": "0x0000000000000000000000000000000000000000",
+                  "tokenGetSymbol": "ETH",
+                  "tokenGetDecimals": 18,
+                  "amountGet": "747000000000000000",
+                  "tokenGive": "0x62a56a4a2ef4d355d34d10fbf837e747504d38d4",
+                  "tokenGiveSymbol": "PAYX",
+                  "tokenGiveDecimals": 2,
+                  "amountGive": "30000",
+                  "nonce": "1982976399",
+                  "expires": 5629999,
+                  "signature": {
+                    "r": "0xfd7aa97d7bdf41ee188ab6db5ce6fcbd312e9f8d1932df9b446820a5a7f6ff4a",
+                    "s": "0x37eab8f9e95629f4ede94ed7e38d14f8fbd6bde6fcc19d24dc2aaea21bd5eaa1",
+                    "v": 28
+                  },
+                  "availableVolume": "747000000000000000",
+                  "created_at": "2018-05-08T21:30:17.399Z",
+                  "updated_at": "2018-05-08T21:30:17.399Z"
+                }
+    post:
```

Comment:
Looks a bit confusing now, so I post on a orders/order_id to do a trade? We already have trades/ endpoint, let's use it?